build-support/tpl_bytecode_handlers_ir_compiler.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+"""
+Usage: tpl_bytecode_handlers_ir_compiler.py LLVM_COMPILER CMAKE_BINARY_DIR BCH_CPP BCH_OUT
```

Comment:
See [this](https://github.com/cmu-db/noisepage/pull/1224/files#diff-af3b638bc2a3e6c650974192a53c7291R500-R505).